src/lightning_lite/strategies/launchers/multiprocessing.py

```diff
@@ -0,0 +1,174 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from dataclasses import dataclass
+from multiprocessing.queues import SimpleQueue
+from typing import Any, Callable, Dict, Optional
+
+import torch
+import torch.backends.cudnn
+import torch.multiprocessing as mp
+from typing_extensions import Literal
+
+from lightning_lite.strategies.launchers.base import _Launcher
+from lightning_lite.utilities.apply_func import move_data_to_device
+from lightning_lite.utilities.imports import _TORCH_GREATER_EQUAL_1_11
+from lightning_lite.utilities.seed import _collect_rng_states, _set_rng_states
+
+
+class _MultiProcessingLauncher(_Launcher):
```

Comment:
Note, the trainer reference and functionality to copy trainer specific attributes (results) to main process has been removed.
No other changes.